athlinks-schema.ttl

```diff
@@ -6,13 +6,24 @@
 @prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 @prefix rdfs: <http://www.w3.org/2000/01/rdf-schema#> .
 @prefix schema: <http://schema.org/> .
-@prefix xsd: <http://www.w3.org/2001/XMLSchema#> .
 
 <http://adp.athlinks.com/2017/01/rdf-schema> a owl:Ontology ;
   owl:imports <http://topbraid.org/schema/> ;
   rdfs:label "The Athlinks Schema vocabulary".
 
 ### CLASSES
+athlinks:TerrainSpecifics a rdfs:Class;
+  rdfs:comment "Specifics about course terrain such as altitude or grade." ;
+  rdfs:label "Terrain Specifics" ;
+  rdfs:subClassOf schema:Landform ;
```

Comment:
Ok, so in Athlinks vr_t_Course table here are the "landform" attributes:
CS_ElevUnit, CS_ElevTypeID, CS_ElevGradeTypeID, CS_TerrainTypeID, CS_Altitude, CS_AltitudeTypeID
None of these point to another table, but are labels or enums.  Which would you recommend we use additionalProperty for, and what should we do with the rest.  These are required to replicate the API calls for event.